tfx/components/testdata/external/README.md

```diff
@@ -13,3 +13,42 @@ df.to_parquet('data.parquet', index=False)
 
 Avro data file can be generated by running the script in
 `avro/generateAvroFile.py`
+
+
+A TF Sequence Example is created under tfrecord_sequence folder. 
```

Comment:
tfrecord_sequence folder contains a tfrecord file of dummy tf.train.SequenceExamples

The format of the testdata is as follows.

**(use code format)**
``` 
context: {
  feature: {
    key  : "i"
    value: {...}
  }
  feature: {
    key  : "f"
    value: {...}
  }
}
feature_lists: {
  feature_list: {
    key  : "s"
    value: {
      feature: {...}
      feature: {...}
    }
  }
}
```